pyHS100/smartbulb.py

```diff
@@ -78,6 +81,20 @@ def is_variable_color_temp(self) -> bool:
         """
         return bool(self.sys_info['is_variable_color_temp'])
 
+    @property
+    def valid_temperature_range(self) -> Tuple[int, int]:
+        """
+        Returns the white temperature range (in Kelvin)
+        depending on the bulb model
+
+        :return: White temperature range in Kelvin (minimun, maximum)
+        :rtype: tuple
+        """
+        for model, temp_range in TPLINK_KELVIN.items():
```

Comment:
I think you should check `is_variable_color_temp` and return (maybe 0, 0) when not available.

Reply:
Setter for temperature could also be adopted to enforce these limits by clamping the values.

Reply:
Do you prefer (0,0) or False ?